Sprint-3/implement/get-angle-type.js

```diff
@@ -25,3 +25,37 @@
 // Identify Reflex Angles:
 // When the angle is greater than 180 degrees and less than 360 degrees,
 // Then the function should return "Reflex angle"
+
+function getAngleType(Angle) {
```

Comment:
Change the parameter name from Angle to angle to adhere to camelCase conventions.